app/views/content/life-as-a-teacher/age-groups-and-specialisms.md

```diff
@@ -12,4 +12,6 @@ content:
   - content/life-as-a-teacher/header
   - content/life-as-a-teacher/age-groups-and-specialisms/categories
   - content/life-as-a-teacher/how-to-become-a-teacher-cta
```

Comment:
Do we need the how to become a teacher CTA as well as the directory? How to become a teacher is already a link in the directory and having this promo as well as the directory feels like a lot?

(this applies to other pages too)

Reply:
agreed - its too much, have removed